lottery/io/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gstd::{prelude::*, ActorId};
+use scale_info::TypeInfo;
+
+#[derive(Debug, Default, Encode, Decode, TypeInfo, Clone, Copy)]
+pub struct LotteryState {
+    pub lottery_started: bool,
+    pub lottery_start_time: u64,
+    pub lottery_duration: u64,
+}
+
+#[derive(Debug, Default, Encode, Decode, TypeInfo, Clone)]
+pub struct Player {
+    pub player_id: ActorId,
+    pub balance: u128,
+}
+
+#[derive(Debug, Decode, Encode, TypeInfo)]
+pub enum Action {
+    ///New player
```

Comment:
Again. Refer to the std Rust doc comments style. And make these comments meaningful or remove them at all.

https://doc.rust-lang.org/src/core/option.rs.html#515-524

```suggestion
    /// Enter the lottery with a new player.
```